modules/tobira/src/main/java/org/opencastproject/tobira/impl/TobiraEndpoint.java

```diff
@@ -204,4 +219,35 @@ private static Response badRequest(String msg) {
     logger.warn("Bad request to tobira/harvest: {}", msg);
     return Response.status(BAD_REQUEST).entity(msg).build();
   }
+
+  @POST
+  @Path("/stats")
+  @Consumes(APPLICATION_JSON)
+  @RestQuery(
+      name = "stats",
+      description = "Accepts a json blob of statistical data about Tobira.",
+      restParameters = {},
+      bodyParameter = @RestParameter(description = "The job blob",
+                                     isRequired = true,
+                                     name = "BODY",
+                                     type = Type.STRING),
+      responses = {
+          @RestResponse(description = "Stats parsed", responseCode = HttpServletResponse.SC_ACCEPTED)
+      },
+      returnDescription = "No data returned, just a 204 on success"
+  )
+  public Response acceptStats(@Context HttpServletRequest request) {
+    try (InputStream is = request.getInputStream()) {
+      String json = IOUtils.toString(is, request.getCharacterEncoding());
+      stats = new Gson().fromJson(json, JsonElement.class).getAsJsonObject();
```

Comment:
I suppose you wanted to use `gson` here? Alternatively, remove the class field.

```suggestion
      stats = gson.fromJson(json, JsonElement.class).getAsJsonObject();
```